### ISSUE TITLE
[Feature] Have a notification widget on the home screen

### ISSUE THREAD

**Issue by SoaibHaque** [MM2-0/Kvaesitso#310]
It would be great if this app have a notification widget on the home screen to see notification without looking at the status bar. The widget have to be more like a invisible widget thats doesn't contain any background color, feel invisible on the screen but any notification comes it shows on the screen right away.

**Comment by vurtomatic** [comment]
Yes, some kind of dynamic notification display on the home screen would be nice.

They could appear in a Notification Widget or as icons under the date in Compact Clock. Tap to open app, long-press for options?

**Comment by mmiradao** [comment]
I really like this idea! To expand on what @SoaibHaque and @vurtomatic suggested, I would love to propose a detailed approach inspired by the "Message Hub" concept (seen in Niagara Launcher / Clicks Communicator). This implementation would integrate seamlessly with Kvaesitso's current design language rather than relying on an invisible or separate widget:

### The Proposal: A Built-in "Message Hub" via the Clock Widget
Currently, the Kvaesitso clock widget has an excellent "fill screen" alignment option (Top, Center, Bottom) and a dynamic info zone controlled by toggles (Media, Date, Alarms, Battery). We could leverage this existing architecture:

*   **Notification Toggle:** Add a new "Notifications" switch to this dynamic area settings.
*   **App Filtering:** Inside its settings, allow users to check/uncheck which specific apps (e.g., WhatsApp, Telegram, Gmail) are allowed to push snippets into this zone.
*   **Dynamic Screen Behavior:** When used alongside the "fill screen" setting, the notifications would dynamically use that empty canvas area. For instance, when aligned to the bottom (or top), the notifications would seamlessly stack and expand the dynamic zone, pushing the other elements accordingly.
*   **Clean State:** When there are no active notifications, the clock area goes back to its clean, minimal layout automatically.

### Expanding the Idea: A Minimalist "Compact/Icon View" Option
To take it a step further and cater to users who prefer an even cleaner layout (similar to what @HalfBloodprinc3 suggested in [issue #515](https://github.com/MM2-0/Kvaesitso/issues/515)), we could introduce a **"Display Mode"** setting inside the Notification toggle:
1.  **Detailed / Snippets Mode:** Shows the active message snippets (the Niagara/Clicks Hub style).
2.  **Compact / Counter Mode:** Shows a minimal row of icons underneath the clock with a numeric counter for pending notifications (e.g., `📞 0  ✉️ 0`), which fits perfectly with the launcher's current aesthetic.

This would be a game-changer for users looking for a clean, privacy-focused notification summary without relying on clunky third-party widgets. Thanks for your amazing work on this project!
